data/schema/Decision_Point.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Decision Point schema definition",
+    "$id": "https://github.com/CERTCC/SSVC/tree/main/data/schema/Decision_Point.schema.json",
```

Comment:
Based on feedback received in #342

Reply:
I pointed these at the raw github repo URL rather than the certcc.github.io/SSVC... url mentioned in https://github.com/CERTCC/SSVC/pull/342#pullrequestreview-1680382681